### ISSUE TITLE
Specify baseNode/baseOffset/extentNode/extentOffset?

### ISSUE THREAD

**Issue by foolip** [w3c/selection-api#34]
setBaseAndExtent as implemented in WebKit/Blink are related to these. Is the plan to drop the attributes but keep the function? Use counter data from Chrome:
https://www.chromestatus.com/metrics/feature/timeline/popularity/400
https://www.chromestatus.com/metrics/feature/timeline/popularity/401
https://www.chromestatus.com/metrics/feature/timeline/popularity/402
https://www.chromestatus.com/metrics/feature/timeline/popularity/403
https://www.chromestatus.com/metrics/feature/timeline/popularity/406

Based on this dropping the attribute looks doable, but is WebKit going to?

**Comment by LoonyBean** [comment]
Should I file this as a non-standard API bug still?

**Comment by rniwa** [comment]
I don't understand what you mean by "non-standard API bug". This is an issue tracker for the selection API. By definition, discussions here will be about standards API.

**Comment by LoonyBean** [comment]
So you are saying baseNode/baseOffset/extentNode/extentOffset are all getting standardized?

Great!
Thanks!

**Comment by rniwa** [comment]
That is precisely what's being discussed now. To decide whether baseNode/baseOffset/extentNode/extentOffset must be standardized or not.

**Comment by foolip** [comment]
@LoonyBean, did you file a Chromium bug to "standardize or remove" these? If so, link to this issue from here, and see if anyone comes along :)

**Comment by LoonyBean** [comment]
I didn't. But I certainly can.

**Comment by jdragonbae** [comment]
I wonder how it's going.
Should we need more discussions?

Thanks :)

**Comment by foolip** [comment]
The current state is that these 4 attributes are in Chrome, Edge and Firefox, but not in Safari. Given that, putting them in the spec certainly seems like the least risky option. However, I'm not sure if there are important differences in behavior that would have to be sorted out.

**Comment by rniwa** [comment]
Huh? These 4 attributes are definitely in Safari since it's a WebKit extension. Did you mean Edge or Firefox instead?

**Comment by foolip** [comment]
Yes, I mean they're in Chrome, Edge and Safari, but not Firefox.
